website/pages/en/billing.mdx

```diff
@@ -2,10 +2,32 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency through Banxa](#billing-on-banxa)
+- [Paying with GRT on Arbitrum](#billing-on-arbitrum)
+
+## Billing on Banxa
+
+Banxa enables you to bypass the need for an exchange and pay for your query fees using the fiat currency of your choice. The fiat currency will be converted to GRT, added to your billing balance on the billing contract, and used to pay for your query fees.
+
+There may be KYC requirements depending on the regulations in your country. For more information about KYC, please visit [Banxa's FAQ page](https://docs.banxa.com/docs/faqs).
+
+You can learn more about Banxa by reading their [documentation](https://docs.banxa.com/docs).
+
+### Paying for query fees with Banxa
+
+1. Select “Pay with Card” option in [Subgraph Studio](https://thegraph.com/studio/).
```

Comment:
You can use a direct link: https://thegraph.com/studio/billing/?show=Deposit (it will be on the Billing page and have the billing pane open)

Reply:
Awesome, that's a lot better! Didn't realize you could do that!